docs/capabilities/transaction-composer.md

```diff
@@ -33,17 +33,146 @@ The [methods to construct a transaction](../code/classes/types_composer.default.
 For example:
 
 ```typescript
+const myMethod = algosdk.ABIMethod.fromSignature('my_method()void')
 const result = algorand
   .newGroup()
   .addPayment({ sender: 'SENDER', receiver: 'RECEIVER', amount: (100).microAlgo() })
   .addAppCallMethodCall({
     sender: 'SENDER',
     appId: 123n,
-    method: abiMethod,
+    method: myMethod,
+    args: [1, 2, 3],
+  })
+```
+
+## Sending a transaction
+
+Once you have constructed all the required transactions, they can be sent by calling `send()` on the `TransactionComposer`.
+Additionally `send()` takes a number of parameters which allow you to opt-in to some additional behaviours as part of sending the transaction or transaction group, mostly significantly `populateAppCallResources` and `coverAppCallInnerTransactionFees`.
+
+### populateAppCallResources
+
+As the name suggests, this setting automatically updates the relevant transactions in the group to include any resources required for the transactions to execute successfully. It leverages the simulate endpoint to discover the accessed resources, which have not been explicitly specified. This setting only applies when you have constucted at least one app call transaction.
```

Comment:
Minor: might be worth giving some examples of what we mean by "resources" given that's quite a general word.